bob.js

```diff
@@ -333,10 +333,26 @@ gameCmd = (message) => {
 
         if (gameName !== game) {
             gameName = game;
+            lastLetter = gameName.charAt(gameName.length - 1);
             if (isILRace()) {
-                levelName = (gameName === "LittleBigPlanet Karting" ? "Karting Lessons" : "Introduction");
+                levelName = (lastLetter === "g" ? "Karting Lessons" : "Introduction");
+                name = levelName;
+            } else if (!(/([\dy]%|\+|al Levels)$/).test(categoryName)) { // That RegExp detects the categories that are common between all games
```

Comment:
I think we should just remove the hard-to-read regex here and always switch to the default category when the game is changed, even if the current category is valid. Still fixes the bug by avoiding invalid categories, but if someone's changing the game I think it's fine to expect them to specify the category as well.